src/large_pages/node_large_page.cc

```diff
@@ -282,20 +294,32 @@ bool IsSuperPagesEnabled() {
 }
 #endif
 
+// Functions in this class must always be inlined because they must end up in
+// the `lpstub` section rather than the `.text` section.
+class MemoryMapPointer {
+ public:
+  FORCE_INLINE MemoryMapPointer(size_t size): size_(size) {}
+  FORCE_INLINE bool operator==(void* rhs) const { return mem_ == rhs; }
+  FORCE_INLINE void* mem() const { return mem_; }
+  MemoryMapPointer(const MemoryMapPointer&) = delete;
+  MemoryMapPointer(MemoryMapPointer&&) = delete;
+  void operator= (const MemoryMapPointer&) = delete;
+  void operator= (const MemoryMapPointer&&) = delete;
```

Comment:
No code is generated for these, right? Otherwise they need to be `FORCE_INLINE` as well.

Reply:
That's right.